### ISSUE TITLE
JAWS does not Read on UIA Focus Call on UIA_CustomControlTypeID

### ISSUE THREAD

**Issue by andreas-stefik** [FreedomScientific/VFO-standards-support#435]
### Summary

> Brief description of the issue. Please list any specific steps.

When JAWS reads a control of type UIA_CustomControlTypeID, and this item gets the focus, JAWS does not read the control. Both NVDA and Narrator correctly read the custom control and my hunch is that JAWS has a flag or something internally to not read it. How to reproduce this is slightly complicated because it's a custom control, but here are the steps:

1. Download Quorum Studio and Install it. Be sure it is version 2.1: https://quorumlanguage.com/download.html

2. Clone a repository from the EPIQ conference using Github: https://github.com/qorf/epiq

3. Inside of Quorum Studio, press CTRL + O to open a project. 

4. Inside the EPIQ folder, go to Scenes/TriggerVoiceOvers/Project/Project.qp. This will open the project.

5. Give it a second and the project will open in Quorum Studio. 

6. There is now a tree in the application. Open up Scenes/Scene.qs in Quorum Studio. This will pop up a scene editor and there will be items in the scene that are UIA_CustomControlTypeID.

For these UIA_CustomControlTypeID, the name field is what I would expect the screen reader to say and this is the default behavior of NVDA and Narrator. These show up correctly in Inspect and Accessibility insights. To be clear, this may not be a bug, per se, it may just be the chosen behavior. But, for highly graphical content you want to be accessible, you need to be able to tell UIA somehow, and custom controls make sense.  

### Expected result

NVDA and Narrator both state the name provided by the custom control. This is the behavior I would have expected.

### Actual result

JAWS makes no sound when the custom control gets focus.

### Example

The steps to reproduce are above. 

#### JAWS version and build number

Latest version on live.
 
#### Operating System and version

Windows 10, 2004, version 19041.450

#### Browser and version:

This is not a browser bug. It appears to be a bug in the UIA implementation in JAWS.

**Comment by stevefaulkner** [comment]
@andreas-stefik This issue tracker is for HTML/ARIA/CSS web UIs only @BenKeyFSI may be able to help you to direct this issue to the appropriate people at Freedom Scientific

> Note: please do not file general support issues here. Please contact Freedom Scientific to report general issues with JAWS.
source: https://github.com/FreedomScientific/VFO-standards-support#vispero-standards-support

**Comment by sinabahram** [comment]
@stevefaulkner  that's helpful information, and sorry for the confusion around filing non-web-related Jaws bugs. Just to confirm, no public bug tracker exists so we will instead reach out to various individuals to try to motivate these internally. These are pretty significant UIA issues, so I'm hoping there is traction because we want to make Jaws better for everyone, not just this accessible programming environment.
